README.md

```diff
@@ -1,63 +1,64 @@
+- ~~Not only does it provide a feature to revisit code, but it also allows developers to roll back the code to a previous state.~~
 - All these features give the developers an environment to experiment and make errors without affecting the final working code.
 - All these updates and experiments can easily be reviewed when using a version control tool.
 
-A version control tool provides many more advantages and powerful features to the developer, but one of the biggest advantages of using this tool is that multiple team members can collaborate on a single project and utilize all these features simultaneously.
+A version control tool provides many more advantages and powerful features to the developer, but one of the biggest advantages of using this tool is that multiple team members can collaborate on a single project and utilize all these features simultaneously ([ref?]()).
 
 Now that we've talked about what a version control system is and its advantages, let's look at how to use Git for version control activities.
-- Step 1 - Create a repository.
-You can create a repository for the code on the GitHub platform by following the simple prompts for creating the repository.
+1. Create a repository.
+You can create a repository for the code on the [GitHub](https://www.github.com) platform by following the simple prompts for creating the repository.
 - Step 2 - Clone the repository.
 Once a repository is created, you need to replicate it to your local machine. You can create files and folders in the repository and start writing your code.
 - Step 3 - Save and commit.
 As you write your code, save your files with every small code block change and commit those changes. This is an important step, as each commit becomes a new state of the code that can later be revisited. It's important to create commits with short but descriptive commit titles and descriptions.
-- Step 4 - View commit history.
-As you develop your project, you can keep track of the history of your commits to ensure the changes you've made to the code.
+- ~~Step 4 - View commit history.~~~
+~~As you develop your project, you can keep track of the history of your commits to ensure the changes you've made to the code.~~
 - Step 5 - Make any required changes.
 If you identify that your code requires changes, revisit those parts, make the changes, and recommit your code.
 - Step 6 - Push changes.
 Finally, all these commits need to be pushed to the remote repository. Once in the cloud, you can access your project from any other device.
 
-(Source:  [Open Source](https://opensource.com/article/18/1/step-step-guide-git)).
+(Source:  [Open Source](https://opensource.com/article/18/1/step-step-guide-git)). _\[I question the accuracy of this reference\]_
```

Comment:
Two comments:
1. The reference/citation you've provided here doesn't quite match your six points above. The reference does overlap somewhat, but it is much more specific to the first three steps, and doesn't really communicate steps 4 - 6 in a meaningful way. So I think this reference is inaccurate.
2. I'm not really sure who these six steps are written for. There isn't enough practical information for a novice programmer, and the steps are quite vague even for a programmer with some experience. Can you do anything to make the steps more meaningful?